### PR TITLE
Scaffolds a new Rails application for the project

### PR DESCRIPTION
**Pull request by dashlykova**
Pivotal Tracker chore description & details [here](https://www.pivotaltracker.com/story/show/167833469)

- Disables the generation of helpers, javascripts, css, and view/ Skips helper, routing and controller specs
- Uses PostgreSQL as database
-Cleans up the gemfile from comments
- Incorporates the following gems: `gem 'bootsnap'`/`gem 'rack-cors'`/`gem 'pry-rails'`/`gem 'pry-byebug'`

### CODE REVIEW

**Comment by dashlykova** [comment]
Pivotal Tracker chore description & details [here](https://www.pivotaltracker.com/story/show/167833469)

- Uses PostgreSQL as database
-Cleans up the gemfile from comments
- Incorporates the following gems: `gem 'bootsnap'`/`gem 'rack-cors'`/`gem 'pry-rails'`/`gem 'pry-byebug'`

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  pry-byebug
+  pry-rails
+  puma (~> 3.11)
+  rack-cors
+  rails (~> 5.2.3)
+  spring
+  spring-watcher-listen (~> 2.0.0)
+
+RUBY VERSION
+   ruby 2.3.4p301
```
Comment (oliverochman) [inline Gemfile.lock]: are you sure that this version of ruby will work with heroku and semaphore?

File: config/application.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module GnosisApi
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.2
+
+    # Settings in config/environments/* take precedence over those specified here.
```
Comment (oliverochman) [inline config/application.rb]: You should add configuration that the application does not generate unnecessary tests.

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 2 above)
Comment (dashlykova) [reply config/application.rb]: Please, review. Have modified the file

File: Gemfile.lock (hunk 1 above)
Comment (dashlykova) [reply Gemfile.lock]: Although ruby 2.3.4 is supported by both heroku and semaphore, I switched ruby version to 2.5.1. In consequence, I had to switch the version of Bundler to the latest version - 2.0.2

Comment (oliverochman) [reply Gemfile.lock]: does bundler 2.0.2 work with semaphore?

File: config/application.rb (hunk 2 above)
Comment (dashlykova) [reply config/application.rb]: Screenshot 2019-08-13 at 14 59 36
Source: [bundler official](https://bundler.io/blog/2018/11/04/an-update-on-bundler-2.html)

**Comment by dashlykova** [comment]
Screenshot 2019-08-13 at 15 03 21
[bundler official](https://bundler.io/blog/2018/11/04/an-update-on-bundler-2.html)

**Review by oliverochman — APPROVED** [review]

**Pull request merged**